src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
```

Comment:
While the constants for price and discounts are used, their names could be more descriptive to reflect their purpose in the context of the car rental domain. For example, `DAILY_RENTAL_RATE`, `DISCOUNT_FOR_3_DAYS`, and `DISCOUNT_FOR_7_DAYS` would make the code more readable and maintainable.